release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1153,6 +1153,9 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * Due to a race condition, the {rh-openstack-first} cloud provider might not start properly. Consequently, LoadBalancer services might never get an `EXTERNAL-IP` set. As a temporary workaround, you can restart the kube-controller-manager pods using the procedure described in link:https://bugzilla.redhat.com/show_bug.cgi?id=2004542[*BZ#2004542*].
 
+* Using the VRF command `ip vrf exec <command>` does not work due to a cgroups mismatch. As a result, this command cannot be used inside OpenShift pods. To use virtual routing and forwarding (VRF), applications must be VRF-aware and bind directly to the VRF interface. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1995631[*BZ#1995631*])
+
+* A non-uniform memory access (NUMA) bug can cause undesired NUMA pinning for the container, which can lead to latency or performance degradation. The Topology Manager can pin the container, under the guaranteed Quality of Service (QoS) pod, with resources that the 'single-numa-node' topology management policy can satisfy to more than one NUMA node. As a workaround, do not start guaranteed QoS pods with container’s memory-resource requests bigger than the 'single-numa-node' policy can provide. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1999603[*BZ#1999603*])
```

Comment:
s/non-uniform/nonuniform
* https://www.merriam-webster.com/dictionary/nonuniform
* see IBMSG 92

The second sentence has a lot of moving parts, and might benefit from being simplified or broken into multiple sentences. After reading through a few times, I can't quite follow what's going on.

Reply:
I also find the "workaround" sentence a bit hard to follow. Is it correct to say something like this?

As a workaround, do not start guaranteed QoS pods when the container memory-resource requests are bigger than the 'single-numa-node' policy can provide.

Reply:
@ctauchen : Thanks for your comment. I have applied 'nonuniform' and split the second sentence in two. 
@bobfuru : Thanks for your comment. I have reworked the 'workaround' sentence as you suggest but just realised it's not in today's version of the .adoc. I have it included now and it will be in the next version of the .adoc in the next days as I have Fixed Issues to include in the Release Notes.